### DISCUSSION TITLE
[draft] "Second Renaissance Match" - a site for finding others working on the second renaissance

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#951] [Ideas]
Already have beginnings of this in the various hangouts.

## Inspiration

Something like RegenMatch for Second Renaissance / Metacrisis

https://www.regenerationpollination.earth

Note: big users of linkedin i suspect https://www.linkedin.com/showcase/regenmatch/about/

Especially idea of networking events

- https://www.regenerationpollination.earth/networking-event

**Comment by asimong** [comment]
Nice thanks for highlighting this Rufus.

**Reply by rufuspollock** [reply]
Thank-you for pointing it out - i should have credited you for the reference but this was a quite copy/paste!
